bin/node-template/runtime/src/lib.rs

```diff
@@ -466,6 +466,24 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_transaction_payment_rpc_runtime_api::TransactionPaymentCallApi<
+		Block,
+		Balance,
+		Call> for Runtime {
```

Comment:
formatting is correct? 🫣

Reply:
could not make rustfmt format this macro body, probably not supported https://github.com/rust-lang/rustfmt/issues/3704
but this time just formatted it outside of macro body with the same indentation.